.generator/templates/api_doc.mustache

```diff
@@ -91,12 +91,13 @@ Name | Type | Description  | Notes
 
 ### Optional Parameters
 
+{{#hasOptionalParams}}
```

Comment:
Should we include the title there, or mention "This endpoint does not have optional params" in case there are none ?